From 9d0b862952339846c15c8d9e759202ddbeefdd44 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Tue, 5 Apr 2016 11:04:30 -0500
Subject: [PATCH] stateless

---
 ipsec.conf | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 ipsec.conf

diff --git a/ipsec.conf b/ipsec.conf
new file mode 100644
index 0000000..c028390
--- /dev/null
+++ b/ipsec.conf
@@ -0,0 +1 @@
+include /etc/ipsec.conf
-- 
2.5.5

