From 709e2a92d768afc6c78e5a243f0076ddec744c8a Mon Sep 17 00:00:00 2001
From: Tuomo Soini <tis@foobar.fi>
Date: Thu, 12 Nov 2015 08:18:18 +0200
Subject: [PATCH] ipsec: custom directory not recognized, github issue #44

---
 programs/ipsec/ipsec.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/programs/ipsec/ipsec.in b/programs/ipsec/ipsec.in
index a022c15..0471228 100755
--- a/programs/ipsec/ipsec.in
+++ b/programs/ipsec/ipsec.in
@@ -239,6 +239,7 @@ case "${1}" in
             else
 		IPSEC_NSSDIR="${2}"
 	    fi
+	    IPSEC_NSSDIR_SQL="sql:${IPSEC_NSSDIR}"
 	fi
 	if [ ! -d "${IPSEC_NSSDIR}" ]; then
 	    mkdir -p "${IPSEC_NSSDIR}"
-- 
2.1.0

